client/lib/user/user.d.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * External Dependencies
+ */
+import { EventEmitter } from 'events';
+import { Language } from '../../languages';
+import { GravatarOptions } from '../../gravatar/types';
+import { URL } from '../../types';
+
+type JSONSerializable = boolean | string | number;
+type WPCOMError = { message: string };
+type WPCOMHeaders = { [ name: string ]: string } & { status: number };
+
+type WPCOMCallback = (
+	error: WPCOMError | null,
+	result: JSONSerializable | null,
+	headers: WPCOMHeaders
+) => void;
+
+export interface User extends EventEmitter {
+	initialize: () => Promise< void >;
+	clearStoreIfChanged: () => void;
+	get: () => UserData;
+	fetch: () => Promise< void >;
+	handleFetchFailure: ( error: Error ) => void;
+	handleFetchSuccess: ( userdata: UserData ) => void;
+	getLanguage: () => Language | void;
+	getAvatarUrl: ( options: GravatarOptions ) => URL;
+	clear: () => Promise< void > | void;
+	sendVerificationEmail< F extends WPCOMCallback >( f: F ): XMLHttpRequest;
+	sendVerificationEmail(): Promise< void >;
+	set: ( attributes: UserData ) => boolean;
+	decrementSiteCount: () => void;
+	incrementSiteCount: () => void;
+	verificationPollerCallback: ( signal: unknown ) => void;
```

Comment:
nitpick: looks like the comment here disappeared across some updates - `( signal?: true )`

Reply:
oh yea oops I'll see if I accidentally reverted anything else..